### PR TITLE
Product Update: API Keys

### PR DESCRIPTION
**Pull request by jayk-u**
Added PU for API Keys
- Visual added
- No help center update required

### CODE REVIEW

File: src/_product-updates/2024-01-06-api-keys [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: API keys
+tags: [Improvement]
+image: /images/changelog/api-keys.jpg
+---
+
+![api-keys.jpg](/images/changelog/api-keys.jpg)
+
+The API key, also known as a token, grants access to our API and allows you to integrate it into your continuous integration workflows.
+Previously specific to a hub or documentation, we are evolving API keys so that they can now operate at the organization level and everything it contains.
```
Comment (Polo2) [inline src/_product-updates/2024-01-06-api-keys:9]: thanks for no mentioning user with apikeys here, they will soon be removed 👍

**Review by Polo2 — APPROVED** [review]
✨ 👍 🔑

**Review by scharrier — COMMENTED** [review]
I think we could add more context here:
* explain precisely what is a scope, and the 3 different levels (API, Hub, global)
* explain why we have added the global level: today, to simplify how they can add new APIs/hubs, tomorrow because we'll add new features to our API which will be out of an API or Hub context (ex: create users, create hubs/apis/branches, etc...)
* add some context about the migration: the existing API/hub keys have been migrated, so it won't break anything in the existing deployment pipelines

**Review by scharrier — APPROVED** [review]
Great, thanks for the changes 👍

**Comment by scharrier** [comment]
@jayk-u it doesn't display on https://docs.bump.sh/product-updates/. Did you test the preview or locally?

**Pull request merged**